fieldservice/views/fsm_location.xml

```diff
@@ -102,11 +102,7 @@
                         <group id="main-left">
                             <field name="id" invisible="1" />
                             <field name="fsm_parent_id" domain="[('id', '!=', id)]" />
-                            <field
-                                name="partner_id"
-                                groups="base.group_no_one"
-                                readonly="1"
-                            />
+                            <field name="partner_id" groups="base.group_no_one" />
```

Comment:
:-1: 

This should go into a separate PR. `partner_id` is also the field used to inherit. Same as res.partner and res.users, it should not be editable by the user.

Reply:
Ah sorry, probably bad rebase. Of course, if I would like to change `partner_id` I would inherit the view.

Reply:
Good catch indeed.